docs/configuration.rst

```diff
@@ -241,14 +241,14 @@ This can be configured in one of two ways.
 **Separate coin sections**
 
 This is an alternative layout for the coin config mentioned above. It provides the ability to change the minloansize per coin, but is otherwise identical in functionality.
-To use this configuration, make sure to comment out the line where coincfg is defined, then add a secion for each coin you wish to configure. These sections should come 
-at the end of the file, after the other options for the bot.
+To use this configuration, make sure to comment out the line where coincfg is defined, then add a section for each coin you wish to configure.
+.. warning:: These sections should come at the end of the file, after the other options for the bot.
```

Comment:
If you install sphinx through pip, you can do `build html` in the `/docs/` directory to build the documentation for you, just make sure not to commit it.